react/features/jwt/middleware.js

```diff
@@ -96,13 +96,7 @@ function _maybeSetCallOverlayVisible({ dispatch, getState }, next, action) {
             default: {
                 // The CallOverlay it to no longer be displayed/visible as soon
                 // as another participant joins.
-                const participants = state['features/base/participants'];
-
-                callOverlayVisible
-                    = Boolean(
-                        participants
-                            && participants.length === 1
-                            && participants[0].local);
+                callOverlayVisible = isLonelyCall(state);
```

Comment:
Not sure about what a "lonely call" is named in the jitsi world but this call is used instead of a direct length check in case there is a shared video as a dummy participant.